### DISCUSSION TITLE
How to make hover and completion popups with borders?

### DISCUSSION THREAD

**Discussion by tokisakiyuu** [LazyVim/LazyVim#4220] [Q&A] [answered]
<img width="691" alt="截屏2024-07-29 21 09 23" src="https://github.com/user-attachments/assets/58ace0b5-8f9d-4a75-9435-d93e416c6451">
<img width="1044" alt="截屏2024-07-29 21 11 25" src="https://github.com/user-attachments/assets/827532af-8be3-4b51-9c5b-56c05ce221ef">

**Comment by dpetka2001** [comment, marked as answer]
Pls take a look at #2708 that has some suggestions.

**Reply by tokisakiyuu** [reply]
Thanks a lot! This is it.
